ocrd_segment/repair.py

```diff
@@ -89,9 +89,20 @@ def process(self):
 
             if plausibilize:
                 new_regions = []
+                reading_order = {}
+                # the reading order does not have to include all regions
+                # but it may include all types of regions!
+                for elem in page.get_ReadingOrder().get_OrderedGroup().get_RegionRefIndexed():
```

Comment:
What about `.get_OrderedGroupIndexed()` here (i.e. recursive reading order)?

Reply:
I cannot find any example of this anywhere in our GT bags or in the PAGE-XML repo.

Reply:
This was definitely missing – although I still do not have an example. See #15.